.travis.yml

```diff
@@ -21,6 +21,10 @@ matrix:
       env: TOXENV=py35
     - python: 3.6
       env: TOXENV=py36
+    - python: 3.7
+      env: TOXENV=py37
+      dist: xenial
+      sudo: true
```

Comment:
Why does this need sudo?

Reply:
According to https://github.com/travis-ci/travis-ci/issues/9815#issuecomment-403657306 it is still required.  Doesn't make sense to me why it is, but hopefully the Travis-CI will fix it so it is not required.